pail/bucket_test.go

```diff
@@ -25,6 +29,67 @@ func newUUID() string {
 	return hex.EncodeToString(b)
 }
 
+func createS3Client(region string) (*s3.S3, error) {
+	sess, err := session.NewSession(&aws.Config{Region: aws.String(region)})
+	if err != nil {
+		return nil, errors.Wrap(err, "problem connecting to AWS")
+	}
+	svc := s3.New(sess)
+	return svc, nil
+}
+
+func createS3Bucket(name, region string) error {
```

Comment:
I think it would be reasonable/good to not create new buckets for the test and instead just use existing buckets. I don't think its reasonable to think that our tests would have keys that could create buckets.